docs/csharp/programming-guide/classes-and-structs/how-to-initialize-objects-by-using-an-object-initializer.md

```diff
@@ -30,6 +30,20 @@ The next example shows the order of execution of constructor and member initiali
 
 :::code language="csharp" source="snippets/object-collection-initializers/ObjectInitializersExecutionOrder.cs" id="ObjectInitializersExecutionOrder":::
 
+## Object initializers without the `new` keyword
+
+You can also use object initializer syntax without the `new` keyword to initialize properties of nested objects. This syntax is particularly useful with read-only properties:
+
+```csharp
+var person = new Person
+{
+    Name = "Alice",
+    Address = { Street = "123 Main St", City = "Anytown" }  // No 'new' keyword needed
+};
+```
```

Comment:
This code example should be expanded to show the types, and should use the snippet you added to this PR with the snippet tag `<SnippetObjectInitializerWithoutNew>`